classes/fetch_services.py

```diff
@@ -81,10 +82,6 @@ def request(self, url: str):
                 f'Could not request URL "{url}" via the YouTube Data API; no API response'
             )
 
-        return response
-
-    def parse(self, response) -> dict:
-        """Parse video data from a YouTube Data API response."""
         if not response["items"]:
             raise VideoUnavailableError(
                 f"Unable to parse response from YouTube Data API; response does not contain any items"
```

Comment:
If this check for response items has been moved out of the `parse` function and into the `request` function, change the error message so that it doesn't refer to parsing.